x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/total_charts.tsx

```diff
@@ -35,62 +27,50 @@ export const TotalCharts: React.FC = () => {
   return (
     <EuiFlexGroup>
       <EuiFlexItem>
-        <EuiPageContent data-test-subj="TotalQueriesChart">
-          <EuiPageContentHeader responsive={false}>
-            <EuiPageContentHeaderSection>
-              <EuiTitle size="xs">
-                <h2>{TOTAL_QUERIES}</h2>
-              </EuiTitle>
-              <EuiText size="s" color="subdued">
-                {LAST_7_DAYS}
-              </EuiText>
-            </EuiPageContentHeaderSection>
-            <EuiPageContentHeaderSection>
-              <EuiButtonTo to={generateEnginePath(ENGINE_ANALYTICS_PATH)} size="s">
-                {VIEW_ANALYTICS}
-              </EuiButtonTo>
-            </EuiPageContentHeaderSection>
-          </EuiPageContentHeader>
-          <EuiPageContentBody>
-            <AnalyticsChart
-              lines={[
-                {
-                  id: TOTAL_QUERIES,
-                  data: convertToChartData({ startDate, data: queriesPerDay }),
-                },
-              ]}
-            />
-          </EuiPageContentBody>
-        </EuiPageContent>
+        <DataPanel
+          data-test-subj="TotalQueriesChart"
+          title={TOTAL_QUERIES}
+          subtitle={LAST_7_DAYS}
+          action={
+            <EuiButtonEmptyTo
+              iconType="eye"
+              to={generateEnginePath(ENGINE_ANALYTICS_PATH)}
+              size="s"
+            >
+              {VIEW_ANALYTICS}
+            </EuiButtonEmptyTo>
+          }
+        >
+          <AnalyticsChart
+            lines={[
+              {
+                id: TOTAL_QUERIES,
+                data: convertToChartData({ startDate, data: queriesPerDay }),
+              },
+            ]}
+          />
+        </DataPanel>
```

Comment:
Uses the new DataPanel component. Eventually (in later updates of Amsterdam) this will support 2 basic versions of panels... a `subdued` version and a `bordered` version. Charts will use the `bordered` version.